src/app/pages/test-report/cat-cpc/components/cpc-end-test-modal/components/modal-result-item/modal-result-item.ts

```diff
@@ -33,6 +36,9 @@ export class ModalResultItemComponent {
 
   displayScore(score: number): string {
     const finalScore = (score || 0);
+    if (this.testState > 0) {
+      return this.adi3 ? (score || 0).toString() : `${finalScore}%`;
+    }
     return this.adi3 ? (score || '-').toString() : `${finalScore}%`;
   }
```

Comment:
Not sure I'm a fan of this seeing as the entire return statement is duplicated to essentially swap the 0 & '-'. Does something like this work?
```
  displayScore(score: number): string {
       if (!this. adi3) {
           const finalScore = (score || 0);
           return `${finalScore}%`;
       }
       return (typeof score === 'number') ? score.toString() : '-';
  }
```